api/android/api/src/androidTest/java/org/nnsuite/nnstreamer/APITestSingleShot.java

```diff
@@ -814,4 +814,35 @@ public void testSNPE() {
             fail();
         }
     }
+
+    @Test
+    public void testSNPEClassificationResult() {
+        if (!NNStreamer.isAvailable(NNStreamer.NNFWType.SNPE)) {
+            /* cannot run the test */
+            return;
+        }
+
+        try {
+            File model = APITestCommon.getSNPEModel();
+
+            SingleShot single = new SingleShot(model, NNStreamer.NNFWType.SNPE);
+
+            /* let's ignore timeout (set 10 sec) */
+            single.setTimeout(10000);
+
+            /* single-shot invoke */
+            TensorsData in = APITestCommon.readRawImageDataSNPE();
+            TensorsData out = single.invoke(in);
+            int labelIndex = APITestCommon.getMaxScoreSNPE(out.getTensorData(0));
+
+            /* check label index (measuring_cup) */
+            if (labelIndex != 648) {
```

Comment:
How about defining `648` as a static variable, like `expected_label`?

Reply:
Thanks. It would be more neat! I update the commit to use final variable `expected_label`.